src/flags.ts

```diff
@@ -44,13 +44,12 @@ export function option<T>(options: {parse: IOptionFlag<T>['parse']} & Partial<IO
 
 const _enum = <T = string>(opts: Parser.flags.EnumFlagOptions<T>): IOptionFlag<T> => {
   return build<T>({
-    parse(input) {
+    parse(input, _) {
```

Comment:
why does parser need a second input?

Reply:
I was just to keep the parser signature, as it takes a second argument. Not really used here.

Removed.